### PR TITLE
Release procedure update

### PR DESCRIPTION
**Pull request by marnovandermaas**
These are some changes that I ran into while doing the v1.0 release.

### CODE REVIEW

File: doc/dev/release-procedure.md [hunk 1]
```diff
@@ -62,10 +62,13 @@ cp sonata-vX.Y.bit.slot3.uf2 ../
 
 ### Automated testing
 
-Run tests on FPGA (note python3.11 or above required), adjust the /dev part to the UART as required.
+Run tests on simulation (note python3.11 or above required). FPGA testing is currently done in CI and requires quite a few add-ons to your board.
```
Comment (GregAC) [inline doc/dev/release-procedure.md:65]: For a release we should definitely repeat the tests on FPGA.

We do need to document the physical add-ons needed to successfully run the tests, though maybe not here as it's got wider applicability.

File: doc/dev/release-procedure.md [hunk 2]
```diff
@@ -62,7 +62,16 @@ cp sonata-vX.Y.bit.slot3.uf2 ../
...
```
Comment (elliotb-lowrisc) [inline doc/dev/release-procedure.md:65]: Nit: multiple sentences on one line

File: doc/dev/release-procedure.md [hunk 3]
```diff
@@ -62,7 +62,16 @@ cp sonata-vX.Y.bit.slot3.uf2 ../
...
+
+```shell
+# Build the simulator
+fusesoc --cores-root=. run --target=sim --tool=verilator --setup --build lowrisc:sonata:system
+# Run the tests
+util/test_runner.py sim -e sw/cheri/build/tests/test_runner --simulator-binary build/lowrisc_sonata_system_0/sim-verilator/Vtop_verilator
+```
+
+Run tests on FPGA (note python3.11 or above required), adjust the /dev part to the UART as required. You will need to connect a Raspberry Pi sense HAT and a temperature sensor to QWIIC 1.
```
Comment (elliotb-lowrisc) [inline doc/dev/release-procedure.md:74]: Nit: multiple sentences on one line

File: doc/dev/release-procedure.md (hunk 2 above)
Comment (marnovandermaas) [reply doc/dev/release-procedure.md]: Thanks this should now be resolved.

File: doc/dev/release-procedure.md (hunk 3 above)
Comment (marnovandermaas) [reply doc/dev/release-procedure.md]: Thanks, this should now be resolved.

**Review by elliotb-lowrisc — APPROVED** [review]
Should probably just get these merged

**Pull request merged**